gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpResponseParser.java

```diff
@@ -39,13 +39,14 @@
 
   /* Parse the http body content JSON stream into the MessageFormatT.
    *
-   * @param httpContent the body of an http response. */
+   * @param httpContent the body of an http response.
```

Comment:
nit: no period per Oracle guidelines

Reply:
Corrected. It seems like we don't have consistency regarding this in our gax library (some docs use dots some don't)